docs/source/array-chunks.rst

```diff
@@ -85,7 +85,7 @@ result in an error.
        x.compute_chunk_sizes()  # for Dask Array
        ddf.to_dask_array(lengths=True)  # for Dask DataFrame ddf
 
-Using :func:`dask.array.Array.compute_chunk_sizes` allows this example run:
+Using ``compute_chunk_sizes`` allows this example run:
```

Comment:
This not working previously means that we likely need to add this to array-api.rst.

Reply:
That's what I thought too. But the end of [array-api.rst][1] has

``` shell
.. currentmodule:: dask.array

Array Methods
~~~~~~~~~~~~~

.. autoclass:: Array
   :members:
```

I'm not seeing `compute_chunk_sizes` in this list:  https://docs.dask.org/en/latest/array-api.html#dask.array.Array

[1]:https://github.com/dask/dask/blob/a4b5a1deb0c910fc5b851f856d64f98639010779/docs/source/array-api.rst

Reply:
Strange. It's present when I build things locally. I can take a closer look later today.

Reply:
The Array.compute_chunk_sizes issue seems to be resolved. I see it at https://docs.dask.org/en/latest/array-api.html#dask.array.Array.compute_chunk_sizes now.

So this change can be reverted.

Reply:
I've reverted most of this. The link with `:func:` or `:meth:` appears as "dask.array.Array.compute_chunk_sizes()" with a link to the docs. I only include that link once each for dataframes and arrays after the example.

Reply:
I don't understand. Why aren't we using :func:\`dask.array.Array.compute_chunk_sizes\`?

Reply:
Nevermind. I've resolved this. Specifically with

``` diff
- :func:`dask.array.Array.compute_chunk_sizes`
+ :func:`~dask.array.Array.compute_chunk_sizes`
```